package.json

```diff
@@ -201,6 +203,7 @@
   },
   "devDependencies": {
     "@actions/core": "1.6.0",
+    "@aws-sdk/client-s3": "3.39.0",
```

Comment:
remove it if it's only a unused peer dependency, the install warning can be ignored

Reply:
ok, removed